### PR TITLE
Feat story details

### PR DESCRIPTION
**Pull request by reginaczech**

### CODE REVIEW

File: client/services/ApiService.tsx [hunk 1]
```diff
@@ -1,29 +1,58 @@
...
+  }
+};
+
+// export const axiosAllProfiles = async (userId: number): Promise<ProfileType[]> => {};
+
+// export const axiosSelectedProfile = async (profileId: number): Promise<ProfileType> => {};
+
+// export const axiosStoriesByProfile = async (profileId: number): Promise<StoryType[]> => {};
+
+export const axiosSelectedStory = async (storyId: number): Promise<StoryType> => {
```
Comment (sebastiandve) [inline client/services/ApiService.tsx:23]: naming is bad, axios should not be part of it

**Review by sebastiandve — CHANGES_REQUESTED** [review]
small tweak

File: client/services/ApiService.tsx
Comment (sebastiandve) [inline client/services/ApiService.tsx:1]: this is not a tsx file just ts

Comment (sebastiandve) [reply client/services/ApiService.tsx:1]: can we rename it to camelCase too

File: client/services/ApiService.tsx (hunk 1 above)
Comment (reginaczech) [reply client/services/ApiService.tsx:23]: Yea i thought so too - ill change that too before pushing

File: client/context/globalContext.tsx
Comment (sebastiandve) [inline client/context/globalContext.tsx:1]: again this is just a ts file not tsx

File: client/apiServices.ts
Comment (sebastiandve) [inline client/apiServices.ts:1]: this file should not exist , move this to the existing file in services

**Review by sebastiandve — CHANGES_REQUESTED** [review]
more tweaks

File: server/router.ts [hunk 2]
```diff
@@ -6,6 +6,8 @@ import { createUser, getUserInfo, getUserProfiles, loginUser } from './controlle
 
 const router = new Router();
 
+//TODO: define ctx.status -> successful post=201 / get=200
+
 router.get('/', (ctx) => {
     ctx.body = 'Hello world!';
   });
```
Comment (sebastiandve) [inline server/router.ts:13]: can we remove this route?

File: client/Types.tsx
Comment (sebastiandve) [inline client/Types.tsx:1]: name of this file should be camelcase too

File: client/context/globalContext.tsx
Comment (reginaczech) [reply client/context/globalContext.tsx:1]: this is a tsx file - it has react components for defining the context provider

File: client/services/apiService.ts [hunk 3]
```diff
@@ -1,17 +1,48 @@
-import axios, {isCancel, AxiosError} from 'axios';
+import { ProfileType, StoryType, UserType } from '../types';
+import axios, { AxiosResponse, isCancel, AxiosError } from 'axios';
 
-const URL = 'http://localhost:3000';
+const BASE_URL = 'http://localhost:3000'; //this may need to change to IP address
 
-async function createStory (
+export const axiosUser = async (id: number): Promise<UserType> => {
```
Comment (sebastiandve) [inline client/services/apiService.ts:6]: axios in the name no good

**Review by sebastiandve — APPROVED** [review]
just one renaming

File: client/services/apiService.ts (hunk 3 above)
Comment (reginaczech) [reply client/services/apiService.ts:6]: good spot

**Pull request merged**